server/implementation/src/main/java/io/smallrye/graphql/execution/context/SmallRyeContext.java

```diff
@@ -199,6 +201,17 @@ public void setDocumentSupplier(DocumentSupplier documentSupplier) {
         this.documentSupplier = documentSupplier;
     }
 
+    public void setExecutionResult(ExecutionResult executionResult) {
+        this.executionResult = executionResult;
+    }
+
+    public Optional<ExecutionResult> getExecutionResult() {
+        if (this.executionResult != null) {
+            return Optional.of(this.executionResult);
+        }
+        return Optional.empty();
+    }
```

Comment:
We do not want to give direct access to graphql-java with a getter (the getters is also in the interface , Context, that will move to MicroProfile spec, and hence can not have any implementation specific things in there. So the way we have done that, is with the unwrap method. So you can remove this, and add a `} else if (wrappedType.equals(ExecutionResult.class)) {` in the unwrap that contains this.

Reply:
Thanks for your review. I could have fixed by dfe5f12

Reply:
Please run a full build locally and commit (see error in CI : `Please format file and commit before running validation!`)